modules/essential_contacts/README.md

```diff
@@ -0,0 +1,52 @@
+# Project API Activation
+
+This optional module is used to enable project APIs in your project. The list of
+APIs to be enabled is specified using the `essential_contacts_config` variable.
+
+This module uses the [`google_essential_contacts_contact`](https://www.terraform.io/docs/providers/google/r/google_project_service.html)
+resource to add contact emails which will receive notification types from Google Cloud, using specified subcription types.
+
+
+## Prerequisites
+
+1. Service account used to run Terraform has permission to administer Essential Contacts:
+[`roles/essentialcontacts.admin`](https://cloud.google.com/iam/docs/understanding-roles#other-roles).
+
+## Example Usage
+```
+module "essential-contacts" {
+  source  = "terraform-google-modules/project-factory/google//modules/essential_contacts"
+  version = "10.1.1"
+
+  project_id                  = "my-project-id"
+
+  essential_contacts_config = [
+    {
+      notification_category_subscriptions = ["ALL"]
+      language_tag              = "en-US"
```

Comment:
WDYT about the config being notification_category_subscriptions=>contacts and language being common for all the contacts?
```tf
essential_contacts_config = {
  "ALL" = [ "user1@foo.com","group1@foo.com"],
  "SECURITY, TECHNICAL" = [ "security@foo.com"],
  "TECHNICAL" = [ "app@foo.com"]
}
language_tag = "en-US"

Reply:
the google_essential_contacts_contact resource is natively set up to accept an array of subscription categories per contact email - which allowed for easy nesting of the configs. The downside to this is that they could not set up multi-language notifications. Perhaps that is an edge case

Reply:
I'd prefer to avoid using comma-delimited lists when we can use native lists instead.

There's probably some room for more brevity in this config though. What if we dispensed with the lists entirely and instead just specified the categories for each user.

```
essential_contacts = {
  "user1@foo.com" = ["ALL"],
  "security@foo.com" = ["SECURITY", "TECHNICAL"],
  "app@foo.com" = ["app@foo.com"]
}
language_tag = "en-US"
```

Reply:
The above approach sgtm @gtsorbo

Reply:
@gtsorbo I think this change was not applied or got lost in the rebase?

Reply:
Looks like it just a readme update thats needed